torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -765,10 +766,13 @@ def perspective_segmentation_mask(img: torch.Tensor, perspective_coeffs: List[fl
 
 def perspective(
     inpt: DType,
-    perspective_coeffs: List[float],
+    startpoints: List[List[int]],
```

Comment:
Here, I missed in previous work that `perspective` mid-level op have to match `F.perspective` from stable one. So, now mid-level `perspective` matches: 
https://github.com/pytorch/vision/blob/3db30442a52e250c569ba131485fc432c8300f16/torchvision/transforms/functional.py#L683-L689